temporian/core/operators/arithmetic.py

```diff
@@ -0,0 +1,186 @@
+        if operation == ArithmeticOperation.DIVISION:
+            return "quot"
+        raise ValueError(f"Unknown operation: {operation}.")
+
+
+class Resolution(str, Enum):
+    PER_FEATURE_IDX = "PER_FEATURE_IDX"
+    PER_FEATURE_NAME = "PER_FEATURE_NAME"
+
+
+class ArithmeticOperator(Operator):
+    """Arithmetic operator."""
+
+    def __init__(
+        self,
+        event_1: Event,
+        event_2: Event,
+        operation: ArithmeticOperation,
+        resolution: Resolution = Resolution.PER_FEATURE_IDX,
+    ):
+        super().__init__()
+
+        # inputs
+        self.add_input("event_1", event_1)
+        self.add_input("event_2", event_2)
+
+        self.add_attribute("operation", operation)
+        self.add_attribute("resolution", resolution)
+
+        if not isinstance(operation, ArithmeticOperation):
+            raise ValueError("operation must be an ArithmeticOperation.")
+
+        if not isinstance(resolution, Resolution):
+            raise ValueError("resolution must be a Resolution.")
+
+        if event_1.sampling() != event_2.sampling():
+            raise ValueError("event_1 and event_2 must have same sampling.")
+
+        if len(event_1.features()) != len(event_2.features()):
+            raise ValueError(
+                "event_1 and event_2 must have same number of features."
+            )
+
+        sampling = event_1.sampling()
+
+        prefix = ArithmeticOperation.prefix(operation)
+
+        # outputs
+        output_features = [  # pylint: disable=g-complex-comprehension
+            Feature(
+                name=f"{prefix}_{event_1_f.name()}_{event_2_f.name()}",
+                dtype=event_1_f.dtype(),
+                sampling=sampling,
+                creator=self,
+            )
+            for event_1_f, event_2_f in zip(
+                event_1.features(), event_2.features()
+            )
+        ]
+
+        self.add_output(
+            "event",
+            Event(
+                features=output_features,
+                sampling=sampling,
+                creator=self,
+            ),
+        )
+        self.check()
+
+    @classmethod
+    def build_op_definition(cls) -> pb.OperatorDef:
+        return pb.OperatorDef(
+            key="ARITHMETIC",
+            attributes=[
+                pb.OperatorDef.Attribute(
+                    key="operation",
+                    type=pb.OperatorDef.Attribute.Type.STRING,
+                    is_optional=False,
+                ),
+                pb.OperatorDef.Attribute(
+                    key="resolution",
+                    type=pb.OperatorDef.Attribute.Type.STRING,
+                    is_optional=False,
+                ),
+            ],
+            inputs=[
+                pb.OperatorDef.Input(key="event_1"),
+                pb.OperatorDef.Input(key="event_2"),
+            ],
+            outputs=[pb.OperatorDef.Output(key="event")],
+        )
+
+
+operator_lib.register_operator(ArithmeticOperator)
+
+
```

Comment:
nit: rename to `sum`, `substract`, `multiply` `divide`?

Reply:
💯